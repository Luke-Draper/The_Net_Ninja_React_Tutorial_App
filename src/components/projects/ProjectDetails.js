import React from "react";

const ProjectDetails = props => {
	const id = props.match.params.id;
	return (
		<div className="container section ProjectDetails">
			<div className="card z-depth-0">
				<div className="card-content">
					<span className="card-title">Project Title : {id}</span>
					<p>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ullam ex
						quae enim soluta cum, consequuntur provident cupiditate tenetur sunt
						dignissimos? Quidem delectus atque quaerat cupiditate quisquam sit
						amet dolore ratione.
					</p>
				</div>
				<div className="card-action grey lighten-4 grey-text">
					<div>Posted by : Luke Draper</div>
					<div>September 2nd, 17:00</div>
				</div>
			</div>
		</div>
	);
};

export default ProjectDetails;
