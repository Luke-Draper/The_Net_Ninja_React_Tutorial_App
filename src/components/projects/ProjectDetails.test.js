import React from "react";
import ProjectDetails from "./ProjectDetails";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("ProjectDetails renders without crashing", () => {
	const props = { match: { params: { id: 1 } } };
	const component = renderer.create(
		<MemoryRouter initialEntries={["/project/1"]}>
			<ProjectDetails {...props} />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
