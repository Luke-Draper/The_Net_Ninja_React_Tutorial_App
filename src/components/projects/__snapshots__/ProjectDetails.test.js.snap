// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectDetails renders without crashing 1`] = `
<div
  className="project-details"
>
  <h3>
    ProjectDetails
  </h3>
</div>
`;
